common-service-core/src/main/java/com/ca/mfaas/message/log/ApimlLogger.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package com.ca.mfaas.message.log;
+
+import com.ca.mfaas.message.core.Message;
+import com.ca.mfaas.message.core.MessageService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Class which allows to control log messages through {@link MessageService}.
+ * <b>Example:</b>
+ * {@code
+ * ApimlLogger logger = ApimlLogger.of(SampleClass.cass, messageService)
+ * }
+ */
+public final class ApimlLogger {
+
+    private final MessageService messageService;
+    private final Logger logger;
+
+    public ApimlLogger(Class<?> clazz, MessageService messageService) {
+        this.messageService = messageService;
+        this.logger = LoggerFactory.getLogger(clazz);
+    }
+
+    /**
+     * Method which allows to create an ApimlLogger object.
+     * @param clazz
```

Comment:
"clazz" or "class"?

Reply:
It is a general practice by Java to name params as "clazz". Because "class" is already taken.
So it is "clazz"

Reply:
uh huh! Good to know :) learn something every day.